etna/models/nn/deepstate/deepstate.py

```diff
@@ -64,6 +70,8 @@ def __init__(
         self.n_samples = n_samples
         self.lr = lr
         self.optimizer_params = {} if optimizer_params is None else optimizer_params
+        self.timestamp_column = timestamp_column
```

Comment:
?

Reply:
Do you suggest we should write logic with `self._timestamp_column` here?

Reply:
Yes, two fields with the same meaning. It's strange

Reply:
I have mixed feelings about it too.
But it could be unintuitive that you set `timestamp_column=None` and then in repr you have `timestamp_column="timestamp"`.
So, it is a more safe way to use some private attribute to this transformation.

Reply:
Create default `timestamp_column="timestamp"`?